### PR TITLE
Ali/self provision of space

### PR DESCRIPTION
**Pull request by ALIRAZA17**
Self Provision of Space

refers this issue: #117

### CODE REVIEW

**Comment by RobinNagpal** [comment]
Tasks - For saving user information
- [ ] remove the login-info page and that information will be now as part of `/new-tidbit-site` page
- [ ] Make sure the tabs on the top show right colors. May be make this tabs as a reusable component
- [ ] Make sure paddings and all are good and the page is finished from styling perspective also
- [ ] Pull latest information of the user and populate it within the first form
- [ ] Make sure validations are enabled

Tasks - Saving of space information 
- We will implement that as part of the new PR. For now concentrate on the above

File: src/app/api/auth/getUser/route.ts
Comment (RobinNagpal) [inline src/app/api/auth/getUser/route.ts:1]: May be the path should be src/app/api/auth/user/route.ts

Comment (ALIRAZA17) [reply src/app/api/auth/getUser/route.ts:1]: done

File: src/components/main/TopNav/TopNav.tsx [hunk 1]
```diff
@@ -147,6 +147,37 @@ export default function TopNav(props: { space?: SpaceWithIntegrationsFragment |
   const { space } = props;
   const router = useRouter();
   const superAdmin = session && isSuperAdmin(session as Session);
+
+  const checkLoginInfo = async () => {
+    router.push('/new-tidbit-site');
+    // try {
```
Comment (RobinNagpal) [inline src/components/main/TopNav/TopNav.tsx:153]: Can you please remove the commented out code

**Pull request merged**